server/src/test/java/org/opensearch/index/mapper/NumberFieldTypeTests.java

```diff
@@ -73,6 +73,7 @@
 import org.opensearch.index.mapper.NumberFieldMapper.NumberType;
 import org.opensearch.index.query.QueryShardContext;
 import org.opensearch.search.MultiValueMode;
+import org.hamcrest.Matchers;
```

Comment:
We can import `org.hamcrest.Matchers.either` only, better than the whole class.

`import static org.hamcrest.Matchers.either;`